integration-tests/tls_test.go

```diff
@@ -1340,6 +1361,14 @@ func getClientSideVerifyConnectionCallback(rootCAs *x509.CertPool) func(cs tls.C
 			}
 		}
 		_, err := cs.PeerCertificates[0].Verify(opts)
+		if err != nil {
+			// use zerolog to avoid interacting with proxy's log messages that are used for assertions in the test
```

Comment:
Just for my understanding, are we suppressing the log messages from a lib to avoid throwing our log assertions off?

Reply:
The goal is to avoid throwing our log assertions off but we aren't suppressing any log messages. We are capturing the log output from the `logrus` library which is what the proxy uses and the test client uses `zerolog`. To avoid messing with the log assertions I decided to use `zerolog` here on this test code in particular instead of `logrus`